x/blob/types/estimate_gas_test.go

```diff
@@ -39,7 +39,7 @@ func TestPFBGasEstimation(t *testing.T) {
 	for idx, tc := range testCases {
 		t.Run(fmt.Sprintf("case %d", idx), func(t *testing.T) {
 			accnts := testfactory.GenerateAccounts(1)
-			testApp, kr := testutil.SetupTestAppWithGenesisValSet(app.DefaultConsensusParams(), accnts...)
+			testApp, kr := testutil.SetupTestAppWithGenesisValSet(app.DefaultInitialConsensusParams(), accnts...)
```

Comment:
note for the reviewer: after i updated DefaultConsensusParams function this test started estimating a tiny bit higher gas, hence why i left it with initial consensus params. 

From the initial look, I'm not sure why updating the app version would result into higher estimates.

Reply:
thoughts? @cmwaters @rootulp @evan-forbes